packages/site/cms-site/src/iframebridge/IFrameBridge.tsx

```diff
@@ -60,6 +61,7 @@ export const IFrameBridgeContext = createContext<IFrameBridgeContext>({
         //empty
     },
     contentScope: undefined,
+    graphQLApiUrl: "",
```

Comment:
That should be undefined by default, and as long as it is undefined the graphlApi can't be used.